tests/topotests/bgp_vrf_lite_best_path_test/bgp_vrf_lite_best_path_topo2.json

```diff
@@ -0,0 +1,1088 @@
+{
+    "address_types": ["ipv4","ipv6"],
+    "ipv4base": "10.0.0.0",
+    "ipv4mask": 24,
+    "ipv6base": "fd00::",
+    "ipv6mask": 64,
+    "link_ip_start": {
+        "ipv4": "10.0.0.0",
+        "v4mask": 24,
+        "ipv6": "fd00::",
+        "v6mask": 64
+    },
+    "lo_prefix": {
+        "ipv4": "1.0.",
+        "v4mask": 32,
+        "ipv6": "2001:db8:f::",
+        "v6mask": 128
+    },
+    "routers": {
+        "r1": {
+            "links": {
+                "r3-link1": {"ipv4": "13.1.1.1/24", "ipv6": "13::1:1/120", "vrf": "RED"},
```

Comment:
Why do we use global IPv4 addresses for documentation? There are specific ranges for documentation: https://datatracker.ietf.org/doc/html/rfc5737 https://datatracker.ietf.org/doc/html/rfc3849

Reply:
> Why do we use global IPv4 addresses for documentation? There are specific ranges for documentation: https://datatracker.ietf.org/doc/html/rfc5737 https://datatracker.ietf.org/doc/html/rfc3849

HI @ton31337 , we are not using these IPs for documentation purpose, this is JSON file which is used for our topology.

Reply:
Topotest is a documentation let's say in this case. We should avoid using someone's IP ranges.

Reply:
> Topotest is a documentation let's say in this case. We should avoid using someone's IP ranges.

Hi @ton31337 , We have updated IPs accordingly. Thank you.